content/styles/03-components/header.css

```diff
@@ -2,69 +2,67 @@ header {
   background-color: var(--white);
   box-shadow: 0 .2rem 0 var(--shadow-color);
   padding: 1rem;
-}
 
-header.main {
-  display: flex;
-  justify-content: space-between;
-  align-items: center;
-  position: fixed;
-  top: 0;
-  left: 0;
-  width: 100%;
-  z-index: 9;
+  &.navigation {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    z-index: 9;
 
-  & h1 {
-    margin: 0;
-    font-weight: 300;
-    font-size: 3rem;
+    & h1 {
+      margin: 0;
+      font-weight: 300;
+      font-size: 3rem;
 
-    & a {
-      padding: 1rem;
-      color: var(--gray);
-      text-decoration: inherit;
+      & a {
+        padding: 1rem;
+        color: var(--gray);
+        text-decoration: inherit;
+      }
     }
-  }
 
-  & > nav {
-    position: relative;
-    margin: 0 2rem 0 0;
-    display: flex;
-    align-items: center;
+    & > nav {
+      position: relative;
+      margin: 0 2rem 0 0;
+      display: flex;
+      align-items: center;
 
-    & a {
-      padding: 3px 15px;
-      font-size: 1.5rem;
-      text-decoration: none;
-    }
+      & a {
+        padding: 3px 15px;
+        font-size: 1.5rem;
+        text-decoration: none;
+      }
 
-    & .button {
-      background-color: var(--primary-bg-color);
-      border: .1rem solid var(--gray);
-      border-radius: .5rem;
+      & .button {
+        background-color: var(--primary-bg-color);
+        border: .1rem solid var(--gray);
+        border-radius: .5rem;
 
-      &:hover {
-        color: var(--black);
-        background-color: var(--gray);
+        &:hover {
+          color: var(--black);
+          background-color: var(--gray);
+        }
       }
     }
-  }
 
-  & .avatar {
-    cursor: pointer;
-  }
+    & .avatar {
+      cursor: pointer;
+    }
 
-  & .nav-account a {
-    display: block;
+    & .nav-account a {
+      display: block;
+    }
   }
-}
 
-header.profile {
-  display: flex;
-  justify-content: space-between;
-  padding: 3rem 1rem;
+  &.profile {
+    padding: 3rem 1rem;
 
-  .profile-socials {
-    margin: 3rem 0;
+    .profile-socials {
```

Comment:
Still wondering about `.profile.profile-socials`.
I can see the risk of calling it .profile.socials` even though it would still make sense.
🤔

Reply:
perhaps something as user-socials ?

Reply:
I changed for `.socials` just tell me if you agree (even though is not a big deal 🤙 )